content/administration/db_management/db_online.rst

```diff
@@ -1,233 +0,0 @@
-
-.. _db_online:
-
-==========================
-Online Database management
-==========================
```

Comment:
Not only it's completly outdated, but it's also not very useful. (before the /my/databases and the database manager were 2 different pages in Odoo Online: now everything is on the same page, and it's quite obvious)

I prefer to remove it, rather than having something wrong. We can still create a new page later if needed.

Reply:
I guess it's just because there are links somewhere to fix; check referrers. Once this link is removed, you won't have trafic to this page. (because the information in this page was totally unuseful and outdated)

Reply:
I confirm that it's used a lot! I can put people on it to update the information. DB management for online or on-premise is the kind of question we have every day at support.